### ISSUE TITLE
Cannot open any url from 10.15.1 (macOS)

### ISSUE THREAD

**Issue by nguyenhuuhanhch** [BurntCaramel/Lantern#6]
Today I have recieved update (Version 1.1.1 (19)). On Catalina I cannot open any url, no open dialog appear.

**Comment by royalicing** [comment]
Are you able to open Console.app and search for Lantern and see any output there?

**Comment by royalicing** [comment]
Are you clicking on the ‘Enter Page URL’ field? If you click you should see:

![image](https://user-images.githubusercontent.com/2635733/68989215-36299480-0897-11ea-9936-595568e1ef8e.png)

**Comment by nguyenhuuhanhch** [comment]
<img width="1083" alt="Screen Shot 2019-11-16 at 4 51 12 PM" src="https://user-images.githubusercontent.com/24221419/68991446-8bfb3e00-0891-11ea-8350-8233dbdd8d83.png">
Please look at screen shot.

**Comment by royalicing** [comment]
Oh wow the URL field is completely missing! I will look into this, hopefully in the next week.

**Comment by thenexus00** [comment]
I went to use it as well today and just blank
![Screenshot 2019-11-21 13 49 44](https://user-images.githubusercontent.com/4312656/69299919-e87da500-0c65-11ea-8847-df2b420e0e5b.png)

**Comment by nguyenhuuhanhch** [comment]
@BurntCaramel have any update about it?

**Comment by royalicing** [comment]
@nguyenhuuhanhch I have tried today and haven’t been able to fix it. I have literally zero idea what is going on :(

I will try on more computers I can get access to.

**Comment by RyanRoberts** [comment]
Same bug here. Such a weird one.

**Comment by royalicing** [comment]
Thanks @RyanRoberts — currently working on moving the URL field out of the toolbar. Will update here once it’s done. Sorry about the wait.

**Comment by royalicing** [comment]
I was able to delete my container (containing preferences etc) and replicate the issue. Version 1.1.3 is out on the Mac App Store and fixes it.

I have changed the toolbar to have a URL field instead of the button + popover. Let me know if there are any issues. Note — I removed the favorites system, which I felt made getting started slower and was more clunky than I wanted. I can focus on adding a new favorites system if anyone would like that (open another issue for this).
